### PR TITLE
Add zapisnice

### PR DESCRIPTION
**Pull request by DeMensa123**

### CODE REVIEW

File: src/zapisnice/zapisnica9_9.tex [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        \end{itemize}
+        \item \textbf {@dena}
+        \begin{itemize}
+            \item text extraction - takmer hotová
+            \item problém pri niektorých typoch súborov
+        \end{itemize}
+        \item \textbf {@karin}
+        \begin{itemize}
+            \item text extraction from images - takmer hotová
+            \item problém pri extrakcii textu
```
Comment (MaliniakovaKarin) [inline src/zapisnice/zapisnica9_9.tex]: change - "problem with import libraries"

File: src/zapisnice/zapisnica9_9.tex [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            \multicolumn{3}{l}{\textbf{Prítomní:}} \\
+            & & & & Bc. František Gič  \\
+            & & & & Bc. Oliver Kanát \\
+            & & & & Bc. Karin Maliniaková \\
+            & & & & Bc. Denisa Mensatorisová \\
+            & & & & Bc. Anton Rusňák \\
+            & & & & Bc. Vladimír Svitok \\
+            & & & & Bc. Kristián Toldy \\
+            \multicolumn{3}{l}{\textbf{Neprítomní:}} & & -\\
+            \multicolumn{3}{l}{\textbf{Zapisovateľ:}} & & Bc. František Gič \\
```
Comment (MaliniakovaKarin) [inline src/zapisnice/zapisnica9_9.tex]: write your name

File: src/zapisnice/zapisnica10.tex [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            & & vedúci: & & Ing. Juraj Petrík \\
+            & & členovia tímu: & & Bc. František Gič  \\
+            & & & & Bc. Oliver Kanát \\
+            & & & & Bc. Karin Maliniaková \\
+            & & & & Bc. Denisa Mensatorisová \\
+            & & & & Bc. Anton Rusňák \\
+            & & & & Bc. Vladimír Svitok \\
+            & & & & Bc. Kristián Toldy \\
+            \multicolumn{3}{l}{\textbf{Neprítomní:}} & & -\\
+            \multicolumn{3}{l}{\textbf{Zapisovateľ:}} & & Bc. František Gič \\
```
Comment (MaliniakovaKarin) [inline src/zapisnice/zapisnica10.tex]: write your name

File: src/zapisnice/zapisnica12.tex [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            & & vedúci: & & Ing. Juraj Petrík \\
+            & & členovia tímu: & & Bc. František Gič  \\
+            & & & & Bc. Oliver Kanát \\
+            & & & & Bc. Karin Maliniaková \\
+            & & & & Bc. Denisa Mensatorisová \\
+            & & & & Bc. Anton Rusňák \\
+            & & & & Bc. Vladimír Svitok \\
+            & & & & Bc. Kristián Toldy \\
+            \multicolumn{3}{l}{\textbf{Neprítomní:}} & & -\\
+            \multicolumn{3}{l}{\textbf{Zapisovateľ:}} & & Bc. František Gič \\
```
Comment (MaliniakovaKarin) [inline src/zapisnice/zapisnica12.tex]: write your name

File: src/zapisnice/zapisnica9.tex [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+            & & vedúci: & & Ing. Juraj Petrík \\
+            & & členovia tímu: & & Bc. František Gič  \\
+            & & & & Bc. Oliver Kanát \\
+            & & & & Bc. Karin Maliniaková \\
+            & & & & Bc. Denisa Mensatorisová \\
+            & & & & Bc. Anton Rusňák \\
+            & & & & Bc. Vladimír Svitok \\
+            & & & & Bc. Kristián Toldy \\
+            \multicolumn{3}{l}{\textbf{Neprítomní:}} & & -\\
+            \multicolumn{3}{l}{\textbf{Zapisovateľ:}} & & Bc. František Gič \\
```
Comment (MaliniakovaKarin) [inline src/zapisnice/zapisnica9.tex]: write your name

**Review by MCFreddie777 — DISMISSED** [review]

**Comment by MCFreddie777** [comment]
Can you please also upload this into the webserver and modify the team web documents page?

**Review by MaliniakovaKarin — DISMISSED** [review]

**Review by MaliniakovaKarin — APPROVED** [review]

**Pull request merged**